scripts/actors/__base.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
This script was copied and pasted three times and each had its own quirks. Here we just unify this script and make sure it works.